docs/docs/manuals/admin-policies.md

```diff
@@ -0,0 +1,75 @@
+---
+title: ZITADEL Policies
+---
+
+Policies are configurations of all the different parts of the IAM. For all parts we have a suitable default in the IAM.
+The default configuration can be overridden for each organization.
+
+## General
+
+You can find these settings in the menu organisation in the section polcies.
+Each policy can be overriden and resetted to the default.
+
+## Password Complexity
+
+With the password complexity policy you can define the requirements for a users password.
+
+The following properties can be set: 
+- Minimum Length
+- Has Uppercase
+- Has Lowercase
+- Has Number
+- Has Symbol
+
+![Password Complexity Policy](/img/manuals/policies/console_org_pw_complexity.png)
+
+## Login Policy
+
+The Login Policy defines how the login process should look like and which authentication options a user has to authenticate.
+
+| Setting | Description |
+| --- | --- |
+| Register allowed | Enable self register possibility in the login ui |
+| Username Password allowed | Possibility to login with username and password |
+| External IDP allowed | Possibility to login with an external identity (e.g Google, Microsoft, Apple, etc)|
+| Force MFA | Force a user to register and use a multifactor authentication |
+| Passwordless | Choose if passwordless login is allowed or not |
+
+![Login Policy](/img/manuals/policies/console_org_login.png)
+
+### Multifactors / Second Factors
+
+In the multifactors section you can configure what kind of multifactors should be allowed.
```

Comment:
```suggestion
In the multifactors section you can configure what kind of multifactors should be allowed. For passwordless U2F (Universial Second Factor) with PIN is required. There is no other option.
```

Reply:
In the multifactors section you can configure what kind of multifactors should be allowed. For passwordless to work, it's required to enable U2F (Universial Second Factor) with PIN. There is no other option at the moment.